LLama.Unittest/LLamaEmbedderTests.cs

```diff
@@ -86,7 +86,7 @@ public async Task EmbedCompareEmbeddingModel()
     [Fact]
     public async Task EmbedCompareGenerateModel()
     {
-        await CompareEmbeddings(Constants.GenerativeModelPath);
+        await CompareEmbeddings(Constants.LLavaModelPath);
```

Comment:
This is meant to be using the generative model (hence the name) not the llava model